konfig.yaml

```diff
@@ -18,6 +18,12 @@ portal:
               path: docs/welcome.md
             - id: concepts
               path: docs/concepts.md
+            - id: Ingest Local Content
```

Comment:
IDs are used as slugs so you probably want something like `ingest-local-content`